src/beforeBuild.ts

```diff
@@ -144,26 +144,53 @@ const BIN_FILE_PATH = join(
   "bin.js",
 );
 
-try {
-  const binFileContent = await readFile(BIN_FILE_PATH, FS_OPTIONS);
-
-  const search = `#!/usr/bin/env bash`;
-  const insert = `export NODE_NO_WARNINGS=1`;
-  const hasSearch = binFileContent.includes(search);
-  const hasInsert = binFileContent.includes(insert);
-
-  if (hasSearch && !hasInsert) {
-    const newBinFileContent = binFileContent.replace(
-      search,
-      `${search}\n${insert}`,
-    );
-
-    await writeFile(BIN_FILE_PATH, newBinFileContent, FS_OPTIONS);
-  } else if (!hasSearch) {
-    throw new Error(`Wasn't able to find '${search}' in ${BIN_FILE_PATH}`);
+const WIN_BIN_FILE_PATH = join(
+  NODE_MODULES_DIR_NAME,
+  "oclif",
+  "lib",
+  "commands",
+  "pack",
+  "win.js",
+);
+
+async function insertContentNextLine(
```

Comment:
I would suggest probably better name that explains what this function does
`patchOclif`